.github/workflows/release.yml

```diff
@@ -22,7 +22,7 @@ jobs:
 
       - name: Make distributables
         run: make -f Makefile.release dist
-      - uses: actions/upload-artifact@v3
+      - uses: actions/upload-artifact@v4
         with:
           name: dist
```

Comment:
I opted to leave this as `dist`, but was tempted to make it `dist-release` for more defined uniqueness. Any input?